examples/timestamping/backend/src/lib.rs

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#![deny(missing_debug_implementations, unsafe_code, bare_trait_objects)]
+//! Timestamping demo. Example of using Exonum framework.
```

Comment:
Timestamping demo. This example shows how to use Exonum framework to create fast and secure service to prove the existence of specific file at some moment of time using blockchain as a secured database.
@MartaDubno?